### DISCUSSION TITLE
Firefly does not open/load

### DISCUSSION THREAD

**Discussion by Buschbanker** [iotaledger/firefly#2338] [Help & Support] [answered]
Hello,

I installed firefly yesterday and created a wallet with legder X. I send iota to the wallet and started staking. If I now want to open firefly to see if everything is in place, I can chose my profile, type in the pin and then nothing happens. I only see the fly shaking its wings on a white underground. In the top left I see the three Menue lines I can choose in the menue but nothing happens.

What can I do?

Would be glad about some help!

Thanks
Buschbanker

**Comment by kraftjs** [comment, marked as answer]
Hi @Buschbanker 

What is your operating system? If you follow this [comment](https://github.com/iotaledger/firefly/issues/2341#issuecomment-1042304593) discussed in Issue #2341, are you able to fix your problem?

**Reply by MetraX88** [reply]
I have the same problem 3 days ago, I created a second profile, which does enter, but my tokens come out at 0, and the transaction is with my 78 iotas.

**Reply by Buschbanker** [reply]
Thanks it worked. I am using Windows. After testing on a second computer. I deleted the Profile and reinstalled everything. No more problems. Thanks!

**Reply by MetraX88** [reply]
You're welcome, it doesn't work for me, I can enter the second profile, but I get 0 iotas, the first profile still doesn't open.

**Reply by MetraX88** [reply]
I can not delete my profile, but I can enter

**Comment by MiKeF05** [comment]
It seems like that users have this problem with the update to the latest version (1.3.3). Maybe this has something to do with the old wallet.rs dependency in the version before. Maybe the update breaks something so a profile recovery has to be done.

**Comment by MetraX88** [comment]
I have the same problem 3 days ago, I created a second profile, which does enter, but my tokens come out at 0, and the transaction is with my 78 iotas.
